app/main.py

```diff
@@ -1,3 +1,48 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f'Distance: {self.km} kilometers.'
+
+    def __repr__(self):
+        return f'Distance(km={self.km})'
+
+    def __add__(self, other):
+        return Distance(self.km + other.km) \
+            if isinstance(other, Distance) else Distance(self.km + other)
```

Comment:
better not use ternary operator in your methods. It is good for returning, for example,  one of two `int` or `string`, not more complex things with comparison or calculations. Fix this in all your methods